Source/WebCore/page/PageConfiguration.h

```diff
@@ -247,6 +251,10 @@ class PageConfiguration {
 #endif
 
     std::optional<MediaSessionManagerFactory> mediaSessionManagerFactory;
+
+#if ENABLE(IMAGE_ANALYSIS)
+    std::optional<ImageTranslationLanguageIdentifiers> shouldDoVisualTranslation;
```

Comment:
Nit - `shouldDoVisualTranslation` seems like a bit of an odd name for a pair of LIDs. Could we instead just name it something like `imageTranslationLanguageIdentifiers` or `languageIdentifiersForVisualTranslation`?